client/blocks/post-share/publicize-actions-list.jsx

```diff
@@ -199,7 +199,7 @@ class PublicizeActionsList extends PureComponent {
 			return <Notice
 				status="is-info"
 				showDismiss={ false }
-				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can!' +
+				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can! ' +
```

Comment:
We should make a8ci18n bot detect these too (and prevent the commit). 
Either way this comment is useless as is.
(todo logged)